saleor/core/utils/random_data.py

```diff
@@ -183,6 +183,7 @@ def create_products(products_data, placeholder_dir, create_images):
         defaults['weight'] = get_weight(defaults['weight'])
         defaults['category_id'] = defaults.pop('category')
         defaults['product_type_id'] = defaults.pop('product_type')
+        defaults['price'] = Money(defaults['price']['amount'], settings.DEFAULT_CURRENCY)
```

Comment:
I believe that we have `Money` instance under the `defaults['price']` so it should look like:
```
defaults['price'] = Money(
    defaults['price'].amount, settings.DEFAULT_CURRENCY)
```